integration-cli/docker_cli_images_test.go

```diff
@@ -241,7 +216,5 @@ func (s *DockerSuite) TestImagesEnsureImagesFromScratchShown(c *check.C) {
 	c.Assert(err, check.IsNil)
 
 	out, _ := dockerCmd(c, "images")
-	if !strings.Contains(out, id[:12]) {
-		c.Fatalf("images should contain images built from scratch (e.g. %s), got %s", id[:12], out)
-	}
+	c.Assert(out, checker.Contains, id[:12], check.Commentf("images should contain images built from scratch (e.g. %s), got %s", id[:12], out))
```

Comment:
No need for `out` here  or the `id[:12]`
I'd probably just move this comment to a normal go comment above the assert.